website/docs/setup.mdx

```diff
@@ -114,16 +147,50 @@ By default, SDK searches for new translation once per application load, but not
 
     Where `interval` - defines translations update time interval in seconds. Minimum allowed interval is 15 minutes (900 seconds).
 
-## Change locale programmatically
+### Change locale programmatically
 
 By default, the SDK relies on the device locale. To change the SDK target language on the fly regardless of the device locale, use the following method:
 
 ```swift
+import CrowdinSDK
+
 CrowdinSDK.enableSDKLocalization(true, localization: "<language_code>")
```

Comment:
This method marked as deprecated. Under the hood it uses CrowdinSDK.currentLocalization.

Reply:
Updated in #316